packages/@react-types/provider/src/index.d.ts

```diff
@@ -77,7 +77,7 @@ export interface ProviderProps extends ContextProps, DOMProps, StyleProps {
   scale?: Scale,
   /**
    * Locale (language specific format) of this provider and its children.
-   * Using the format primary-region, ex. en-US, fr-CA, ar-AE.
+   * Uses the format primary-region, e.g. En-US, fr-CA, ar-AE.
```

Comment:
I assume the linter picked up the `en-US` as the start of a new sentence here?

Reply:
oo, good catch, i'll make sure it doesn't force that